,test_name,code_len,dumb_dyn_slice,dyn_slice,pruned_slice,dumb_dyn_slice_len,pruned_slice_len,dyn_slice_len,runtime_bare_test,runtime_tracing_augmented_test,runtime_dyn_slice,runtime_pruned_slice,perf_runtime_bare_test,perf_runtime_tracing_augmented_test,perf_runtime_dyn_slice,perf_runtime_pruned_slice,len_exec_trace,test_result,sliced_result_equal_to_bare,exception,stage,num_boolops_pruned_sliced,num_lines_with_boolops_pruned_sliced,num_boolops_dyn_sliced,num_lines_with_boolops_dyn_sliced,num_boolops,num_lines_with_boolops,benchmark,test_status
67,benchmark/quixbugs/correct_python_programs/hanoi.py#1:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'helper' is not defined"",)",run sliced,,,,,0,0,quixbugs,Other Exception
12983,benchmark/refactory/data/question_2/code/fail/wrong_2_107.py#0:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
12984,benchmark/refactory/data/question_2/code/fail/wrong_2_107.py#1:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
12985,benchmark/refactory/data/question_2/code/fail/wrong_2_107.py#2:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
12986,benchmark/refactory/data/question_2/code/fail/wrong_2_107.py#3:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
12987,benchmark/refactory/data/question_2/code/fail/wrong_2_107.py#4:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
12988,benchmark/refactory/data/question_2/code/fail/wrong_2_107.py#5:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
12989,benchmark/refactory/data/question_2/code/fail/wrong_2_107.py#6:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
12990,benchmark/refactory/data/question_2/code/fail/wrong_2_107.py#7:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
12991,benchmark/refactory/data/question_2/code/fail/wrong_2_107.py#8:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
12992,benchmark/refactory/data/question_2/code/fail/wrong_2_107.py#9:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
12993,benchmark/refactory/data/question_2/code/fail/wrong_2_107.py#10:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
12994,benchmark/refactory/data/question_2/code/fail/wrong_2_107.py#11:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
12995,benchmark/refactory/data/question_2/code/fail/wrong_2_107.py#12:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
12996,benchmark/refactory/data/question_2/code/fail/wrong_2_107.py#13:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
12997,benchmark/refactory/data/question_2/code/fail/wrong_2_107.py#14:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
12998,benchmark/refactory/data/question_2/code/fail/wrong_2_107.py#15:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
12999,benchmark/refactory/data/question_2/code/fail/wrong_2_107.py#16:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13000,benchmark/refactory/data/question_2/code/fail/wrong_2_108.py#0:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13001,benchmark/refactory/data/question_2/code/fail/wrong_2_108.py#1:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13002,benchmark/refactory/data/question_2/code/fail/wrong_2_108.py#2:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13003,benchmark/refactory/data/question_2/code/fail/wrong_2_108.py#3:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13004,benchmark/refactory/data/question_2/code/fail/wrong_2_108.py#4:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13005,benchmark/refactory/data/question_2/code/fail/wrong_2_108.py#5:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13006,benchmark/refactory/data/question_2/code/fail/wrong_2_108.py#6:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13007,benchmark/refactory/data/question_2/code/fail/wrong_2_108.py#7:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13008,benchmark/refactory/data/question_2/code/fail/wrong_2_108.py#8:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13009,benchmark/refactory/data/question_2/code/fail/wrong_2_108.py#9:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13010,benchmark/refactory/data/question_2/code/fail/wrong_2_108.py#10:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13011,benchmark/refactory/data/question_2/code/fail/wrong_2_108.py#11:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13012,benchmark/refactory/data/question_2/code/fail/wrong_2_108.py#12:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13013,benchmark/refactory/data/question_2/code/fail/wrong_2_108.py#13:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13014,benchmark/refactory/data/question_2/code/fail/wrong_2_108.py#14:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13015,benchmark/refactory/data/question_2/code/fail/wrong_2_108.py#15:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13016,benchmark/refactory/data/question_2/code/fail/wrong_2_108.py#16:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13017,benchmark/refactory/data/question_2/code/fail/wrong_2_109.py#0:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13018,benchmark/refactory/data/question_2/code/fail/wrong_2_109.py#1:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13019,benchmark/refactory/data/question_2/code/fail/wrong_2_109.py#2:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13020,benchmark/refactory/data/question_2/code/fail/wrong_2_109.py#3:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13021,benchmark/refactory/data/question_2/code/fail/wrong_2_109.py#4:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13022,benchmark/refactory/data/question_2/code/fail/wrong_2_109.py#5:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13023,benchmark/refactory/data/question_2/code/fail/wrong_2_109.py#6:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13024,benchmark/refactory/data/question_2/code/fail/wrong_2_109.py#7:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13025,benchmark/refactory/data/question_2/code/fail/wrong_2_109.py#8:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13026,benchmark/refactory/data/question_2/code/fail/wrong_2_109.py#9:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13027,benchmark/refactory/data/question_2/code/fail/wrong_2_109.py#10:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13028,benchmark/refactory/data/question_2/code/fail/wrong_2_109.py#11:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13029,benchmark/refactory/data/question_2/code/fail/wrong_2_109.py#12:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13030,benchmark/refactory/data/question_2/code/fail/wrong_2_109.py#13:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13031,benchmark/refactory/data/question_2/code/fail/wrong_2_109.py#14:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13032,benchmark/refactory/data/question_2/code/fail/wrong_2_109.py#15:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
13033,benchmark/refactory/data/question_2/code/fail/wrong_2_109.py#16:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15516,benchmark/refactory/data/question_2/code/wrong/wrong_2_107.py#0:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15517,benchmark/refactory/data/question_2/code/wrong/wrong_2_107.py#1:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15518,benchmark/refactory/data/question_2/code/wrong/wrong_2_107.py#2:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15519,benchmark/refactory/data/question_2/code/wrong/wrong_2_107.py#3:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15520,benchmark/refactory/data/question_2/code/wrong/wrong_2_107.py#4:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15521,benchmark/refactory/data/question_2/code/wrong/wrong_2_107.py#5:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15522,benchmark/refactory/data/question_2/code/wrong/wrong_2_107.py#6:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15523,benchmark/refactory/data/question_2/code/wrong/wrong_2_107.py#7:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15524,benchmark/refactory/data/question_2/code/wrong/wrong_2_107.py#8:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15525,benchmark/refactory/data/question_2/code/wrong/wrong_2_107.py#9:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15526,benchmark/refactory/data/question_2/code/wrong/wrong_2_107.py#10:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15527,benchmark/refactory/data/question_2/code/wrong/wrong_2_107.py#11:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15528,benchmark/refactory/data/question_2/code/wrong/wrong_2_107.py#12:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15529,benchmark/refactory/data/question_2/code/wrong/wrong_2_107.py#13:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15530,benchmark/refactory/data/question_2/code/wrong/wrong_2_107.py#14:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15531,benchmark/refactory/data/question_2/code/wrong/wrong_2_107.py#15:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15532,benchmark/refactory/data/question_2/code/wrong/wrong_2_107.py#16:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15533,benchmark/refactory/data/question_2/code/wrong/wrong_2_108.py#0:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15534,benchmark/refactory/data/question_2/code/wrong/wrong_2_108.py#1:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15535,benchmark/refactory/data/question_2/code/wrong/wrong_2_108.py#2:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15536,benchmark/refactory/data/question_2/code/wrong/wrong_2_108.py#3:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15537,benchmark/refactory/data/question_2/code/wrong/wrong_2_108.py#4:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15538,benchmark/refactory/data/question_2/code/wrong/wrong_2_108.py#5:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15539,benchmark/refactory/data/question_2/code/wrong/wrong_2_108.py#6:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15540,benchmark/refactory/data/question_2/code/wrong/wrong_2_108.py#7:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15541,benchmark/refactory/data/question_2/code/wrong/wrong_2_108.py#8:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15542,benchmark/refactory/data/question_2/code/wrong/wrong_2_108.py#9:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15543,benchmark/refactory/data/question_2/code/wrong/wrong_2_108.py#10:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15544,benchmark/refactory/data/question_2/code/wrong/wrong_2_108.py#11:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15545,benchmark/refactory/data/question_2/code/wrong/wrong_2_108.py#12:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15546,benchmark/refactory/data/question_2/code/wrong/wrong_2_108.py#13:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15547,benchmark/refactory/data/question_2/code/wrong/wrong_2_108.py#14:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15548,benchmark/refactory/data/question_2/code/wrong/wrong_2_108.py#15:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15549,benchmark/refactory/data/question_2/code/wrong/wrong_2_108.py#16:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15550,benchmark/refactory/data/question_2/code/wrong/wrong_2_109.py#0:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15551,benchmark/refactory/data/question_2/code/wrong/wrong_2_109.py#1:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15552,benchmark/refactory/data/question_2/code/wrong/wrong_2_109.py#2:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15553,benchmark/refactory/data/question_2/code/wrong/wrong_2_109.py#3:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15554,benchmark/refactory/data/question_2/code/wrong/wrong_2_109.py#4:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15555,benchmark/refactory/data/question_2/code/wrong/wrong_2_109.py#5:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15556,benchmark/refactory/data/question_2/code/wrong/wrong_2_109.py#6:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15557,benchmark/refactory/data/question_2/code/wrong/wrong_2_109.py#7:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15558,benchmark/refactory/data/question_2/code/wrong/wrong_2_109.py#8:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15559,benchmark/refactory/data/question_2/code/wrong/wrong_2_109.py#9:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15560,benchmark/refactory/data/question_2/code/wrong/wrong_2_109.py#10:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15561,benchmark/refactory/data/question_2/code/wrong/wrong_2_109.py#11:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15562,benchmark/refactory/data/question_2/code/wrong/wrong_2_109.py#12:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15563,benchmark/refactory/data/question_2/code/wrong/wrong_2_109.py#13:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15564,benchmark/refactory/data/question_2/code/wrong/wrong_2_109.py#14:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15565,benchmark/refactory/data/question_2/code/wrong/wrong_2_109.py#15:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
15566,benchmark/refactory/data/question_2/code/wrong/wrong_2_109.py#16:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'count_dates' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
22102,benchmark/refactory/data/question_3/code/wrong/wrong_3_079.py#3:,,,,,,,,,,,,,,,,,,,"<class 'ValueError'>('list.remove(x): x not in list',)",run sliced,,,,,0,0,refactory,Other Exception
22103,benchmark/refactory/data/question_3/code/wrong/wrong_3_079.py#4:,,,,,,,,,,,,,,,,,,,"<class 'ValueError'>('list.remove(x): x not in list',)",run sliced,,,,,0,0,refactory,Other Exception
22104,benchmark/refactory/data/question_3/code/wrong/wrong_3_079.py#5:,,,,,,,,,,,,,,,,,,,"<class 'ValueError'>('list.remove(x): x not in list',)",run sliced,,,,,0,0,refactory,Other Exception
23737,benchmark/refactory/data/question_4/code/correct/correct_4_285.py#0:,,,,,,,,,,,,,,,,,,,"<class 'IndexError'>('list index out of range',)",run sliced,,,,,0,0,refactory,Other Exception
23738,benchmark/refactory/data/question_4/code/correct/correct_4_285.py#1:,,,,,,,,,,,,,,,,,,,"<class 'IndexError'>('list index out of range',)",run sliced,,,,,0,0,refactory,Other Exception
23739,benchmark/refactory/data/question_4/code/correct/correct_4_285.py#2:,,,,,,,,,,,,,,,,,,,"<class 'IndexError'>('list index out of range',)",run sliced,,,,,0,0,refactory,Other Exception
23740,benchmark/refactory/data/question_4/code/correct/correct_4_285.py#3:,,,,,,,,,,,,,,,,,,,"<class 'IndexError'>('list index out of range',)",run sliced,,,,,0,0,refactory,Other Exception
23741,benchmark/refactory/data/question_4/code/correct/correct_4_285.py#4:,,,,,,,,,,,,,,,,,,,"<class 'IndexError'>('list index out of range',)",run sliced,,,,,0,0,refactory,Other Exception
23742,benchmark/refactory/data/question_4/code/correct/correct_4_285.py#5:,,,,,,,,,,,,,,,,,,,"<class 'IndexError'>('list index out of range',)",run sliced,,,,,0,0,refactory,Other Exception
25801,benchmark/refactory/data/question_4/code/wrong/wrong_4_151.py#0:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'new' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
25802,benchmark/refactory/data/question_4/code/wrong/wrong_4_151.py#1:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'new' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
25803,benchmark/refactory/data/question_4/code/wrong/wrong_4_151.py#2:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'new' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
25804,benchmark/refactory/data/question_4/code/wrong/wrong_4_151.py#3:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'new' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
25805,benchmark/refactory/data/question_4/code/wrong/wrong_4_151.py#4:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'new' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
25806,benchmark/refactory/data/question_4/code/wrong/wrong_4_151.py#5:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'new' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
25819,benchmark/refactory/data/question_4/code/wrong/wrong_4_154.py#0:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'new' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
25820,benchmark/refactory/data/question_4/code/wrong/wrong_4_154.py#1:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'new' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
25821,benchmark/refactory/data/question_4/code/wrong/wrong_4_154.py#2:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'new' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
25822,benchmark/refactory/data/question_4/code/wrong/wrong_4_154.py#3:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'new' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
25823,benchmark/refactory/data/question_4/code/wrong/wrong_4_154.py#4:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'new' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
25824,benchmark/refactory/data/question_4/code/wrong/wrong_4_154.py#5:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'new' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
25885,benchmark/refactory/data/question_4/code/wrong/wrong_4_170.py#0:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'sorted_lst' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
25886,benchmark/refactory/data/question_4/code/wrong/wrong_4_170.py#1:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'sorted_lst' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
25887,benchmark/refactory/data/question_4/code/wrong/wrong_4_170.py#2:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'sorted_lst' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
25888,benchmark/refactory/data/question_4/code/wrong/wrong_4_170.py#3:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'sorted_lst' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
25889,benchmark/refactory/data/question_4/code/wrong/wrong_4_170.py#4:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'sorted_lst' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
25890,benchmark/refactory/data/question_4/code/wrong/wrong_4_170.py#5:,,,,,,,,,,,,,,,,,,,"<class 'NameError'>(""name 'sorted_lst' is not defined"",)",run sliced,,,,,0,0,refactory,Other Exception
26077,benchmark/refactory/data/question_4/code/wrong/wrong_4_222.py#0:,,,,,,,,,,,,,,,,,,,"<class 'IndexError'>('list index out of range',)",run sliced,,,,,0,0,refactory,Other Exception
26078,benchmark/refactory/data/question_4/code/wrong/wrong_4_222.py#1:,,,,,,,,,,,,,,,,,,,"<class 'IndexError'>('list index out of range',)",run sliced,,,,,0,0,refactory,Other Exception
26079,benchmark/refactory/data/question_4/code/wrong/wrong_4_222.py#2:,,,,,,,,,,,,,,,,,,,"<class 'IndexError'>('list index out of range',)",run sliced,,,,,0,0,refactory,Other Exception
26080,benchmark/refactory/data/question_4/code/wrong/wrong_4_222.py#3:,,,,,,,,,,,,,,,,,,,"<class 'IndexError'>('list index out of range',)",run sliced,,,,,0,0,refactory,Other Exception
26081,benchmark/refactory/data/question_4/code/wrong/wrong_4_222.py#4:,,,,,,,,,,,,,,,,,,,"<class 'IndexError'>('list index out of range',)",run sliced,,,,,0,0,refactory,Other Exception
26082,benchmark/refactory/data/question_4/code/wrong/wrong_4_222.py#5:,,,,,,,,,,,,,,,,,,,"<class 'IndexError'>('list index out of range',)",run sliced,,,,,0,0,refactory,Other Exception
26455,benchmark/refactory/data/question_4/code/wrong/wrong_4_289.py#0:,,,,,,,,,,,,,,,,,,,"<class 'UnboundLocalError'>(""local variable 'new_list' referenced before assignment"",)",run sliced,,,,,0,0,refactory,Other Exception
26456,benchmark/refactory/data/question_4/code/wrong/wrong_4_289.py#1:,,,,,,,,,,,,,,,,,,,"<class 'UnboundLocalError'>(""local variable 'new_list' referenced before assignment"",)",run sliced,,,,,0,0,refactory,Other Exception
26457,benchmark/refactory/data/question_4/code/wrong/wrong_4_289.py#2:,,,,,,,,,,,,,,,,,,,"<class 'UnboundLocalError'>(""local variable 'new_list' referenced before assignment"",)",run sliced,,,,,0,0,refactory,Other Exception
26458,benchmark/refactory/data/question_4/code/wrong/wrong_4_289.py#3:,,,,,,,,,,,,,,,,,,,"<class 'UnboundLocalError'>(""local variable 'new_list' referenced before assignment"",)",run sliced,,,,,0,0,refactory,Other Exception
26459,benchmark/refactory/data/question_4/code/wrong/wrong_4_289.py#4:,,,,,,,,,,,,,,,,,,,"<class 'UnboundLocalError'>(""local variable 'new_list' referenced before assignment"",)",run sliced,,,,,0,0,refactory,Other Exception
26460,benchmark/refactory/data/question_4/code/wrong/wrong_4_289.py#5:,,,,,,,,,,,,,,,,,,,"<class 'UnboundLocalError'>(""local variable 'new_list' referenced before assignment"",)",run sliced,,,,,0,0,refactory,Other Exception
